components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/jwks/JwksEndpoint.java

```diff
@@ -114,8 +114,8 @@ public String jwks() {
             jwksKeyArray.put(jwksKeys);
             jwksJson.put("keys", jwksKeyArray);
         } catch (Exception e) {
-            String errorMesage = "Error while generating the keyset";
-            log.error(errorMesage);
+            String errorMesage = "Error while generating the keyset for " + tenantDomain + " tenant domain.";
```

Comment:
Can be improved as "Error while generating the keyset for tenant domain: " + tenantDomain

Reply:
Fixed in master